src/moodselector.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="Update your description here" />
+	<title>mood selector page</title>
+</head>
+<body>
+	<div class="topnav">
+		<a href="index.html">home</a>
+		<a class="active" href="moodselector.html">mood selector</a>
+		<a href="results.html">your music</a>
+		<a href="aboutus.html">about us</a>
+	</div>
+	<div class="content-container">
+		<p class="page-tab">Survey</p>
+		<h3 class="page-heading">Please answer the following questions:</h3>
+			<p class="question-1">Age</p>
+			<p class="question-2">Preferred Genre</p>
+			<p class="question-3">Mood</p>
+			<p class="question-4">Music Decade</p>
+			<a class="button-results">See My Results</a>
```

Comment:
Consider using a `<button>` tag here instead, since this is an interactive element. https://medium.com/design-code-repository/a-vs-button-b859547cae4d